modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -460,7 +460,7 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
       values: List[EnumValue],
       hints: List[Hint]
   ): RenderResult = lines(
-    s"sealed abstract class $name(val value: String, val ordinal: Int) extends Product with Serializable",
+    s"sealed abstract class $name(val value: String, val ordinal: Int) extends scala.Product with scala.Serializable",
```

Comment:
general thought, should we do the same for collections? I imagine a type like `Vector` might appear in users' smithy files

Reply:
I don't think that should be default behaviour, but maybe we'll add that as an option to the rendering